engine/caches.go

```diff
@@ -56,6 +56,7 @@ var precachedPartitions = utils.StringMap{
 	utils.CacheChargerProfiles:     true,
 	utils.CacheDispatcherProfiles:  true,
 	utils.CacheDispatcherHosts:     true,
+	utils.CacheRPCConnections:      true,
```

Comment:
Not sure I remember here, do we pre-cache RPCConnections or what is the purpose here?